src/tiles/list/tile-list-compact.ts

```diff
@@ -63,11 +65,14 @@ export class TileListCompact extends LitElement {
     if (!this.model?.identifier) {
       return nothing;
     }
-    return html` <img
-      src="${this.baseImageUrl}/services/img/${this.model.identifier}"
-      alt="${this.model.identifier}"
-      class="${this.model?.mediatype}"
-    />`;
+    return html`
+      <item-image
+        .model=${this.model}
+        .baseImageUrl=${this.baseImageUrl}
+        .isListTile=${true}
+      >
```

Comment:
QUESTION: Is it desired to have a unit of whitespace between the opening and closing `item-image` tags?

Reply:
The indentation and end tag on a separate line following examples in lit.dev and @jbuckner's code.
I like it for readability.
I have found whitespace problems in HTML display only when a component wraps another component.